frb_codegen/src/binary/commands.rs

```diff
@@ -145,6 +145,9 @@ pub(crate) struct GenerateCommandArgsPrimary {
     #[arg(long)]
     pub full_dep: bool,
 
+    /// Configure rust opaque implementation, "moi" or "nom"
+    pub rust_opaque: Option<String>,
```

Comment:
nit:

1. Maybe name it default_rust_opaque_codec
2. maybe make it type `Option<RustOpaqueCodecMode>`